README.md

```diff
@@ -74,25 +82,119 @@ _NOTE_: the above `docker` command line contains the oprthomosaic file without i
 **Results:**
 Upon a successful run the output will contain one sub-folder for each plot.
 In the sub-folders for plots that intersected with the orthomosaic, there will be the clipped `.tif` file and two `.csv` files.
-The CSV files both contain much of the same information.
+This will generate one directory per plot in the `outputs/` folder.
+Each plot will contain two key outputs of interest:
+1. `orthomosaic_mask.tif`
+2. `canopycover.csv` with the canopy cover calculated from the mask file
+   * [In the future](https://github.com/AgPipeline/issues-and-projects/issues/210), these CSV files will be aggregated into a single file for each run.
 The file with "geostreams" in its name can be uploaded to TERRAREF's Geostreams database.  
 
+### Canopy Cover: OpenDroneMap and plot boundaries <a name="opendm_can_shp" />
+
+The following steps are used when wanting to use OpenDroneMap (ODM) to create the Orthomosaic image that's then used to create the canopy cover values.
+As with the [previous example](#om_can_shp) we will be listing the steps and then providing an example.
+
+_NOTE_: the SciF Docker image uses Docker sibling containers to run the OpenDroneMap application.
+Please read our section on [Docker Sibling Containers](#docker_sibling_containers) below to be informed of potential risks with this approach.
+
+1. Create two named Docker volumes to use for processing data; one for input files and one for output files - the same volume can be used for both if desired
+2. Copy the source drone images into a folder
+3. If using a [shapefile](#shapefile) or [GeoJSON](#geojson) file, copy those into the same folder as the drone images
+4. Copy the experiment metadata file into the same folder as the drone images
+5. Copy the folder contents of the drone images folder that was just prepared onto the input named volume
+6. Create another folder for the output folders and files
+7. Run the docker container's `odm_workflow` app specifying  either the name of the shapefile or geojson file, or the URL of they [BETYdb](#betydb) instance to query for plot boundaries, and the two named volumes
+8. Copy the resulting files off the output named volume to the local folder
+9. Clean up the named volumes
+
+#### For example: <a name="opendm_can_shp_example" />
+
+You can download a sample dataset of files (archived) with names corresponding to those listed here from CyVerse using the following command.
+```bash
+curl -X GET https://de.cyverse.org/dl/d/7D28E988-67A2-498A-B18C-E0D884FD0C83/scif_odm_test_data.tar.gz > scif_odm_test_data.tar.gz
+tar xvzf scif_test_data.tar.gz -C ${PWD}/inputs
```

Comment:
```suggestion
tar xvzf scif_test_data.tar.gz -C ${PWD}/inputs
mv inputs/IMG/* inputs/
```

* any reason not to structure the archive in the same way it will be used? i.e. either omit `IMG/` or have the workflow look there?

Reply:
@dlebauer This is a more realistic way that the data will be available: Drone captures are usually in a separate folder

Reply:
The workflow is determined by what Open Drone Maps wants things organized and this is a simple way for the user to specify the source files they want for processing

Reply:
> This is a more realistic way that the data will be available: 

definitely a good idea to stick w/ the default of how people will have it.